src/main/js/App.js

```diff
@@ -1,11 +1,259 @@
+                    size="small"
+                    onChange={e => setSurveyName(e.target.value)}
+                />
+                <Button className="qq-app m" variant="contained" color="primary" onClick={createSurvey}>Create</Button>
+            </div>
+            <div>
+                <FormControl className="qq-app mv qq-app__qtype_select">
+                    <InputLabel id="qtype_select_label">Question Type</InputLabel>
+                    <Select labelId="qtype_select_label" value={currentType} onChange={e => setCurrentType(e.target.value)}>
+                        <MenuItem value={qType.OPEN_ENDED}>Open-Ended</MenuItem>
+                        <MenuItem value={qType.NUMERICAL}>Numerical</MenuItem>
+                    </Select>
+                </FormControl>
+                <Button className="qq-app m" variant="contained" color="primary" disabled={!currentType} onClick={addQuestion}>+</Button>
+            </div>
+            <div>
+                <Button className="qq-app mv" variant="contained" color="primary" onClick={myFunc1}>Dev Test 1 (POST)</Button>
+                <Button className="qq-app m" variant="contained" color="primary" onClick={myFunc2}>Dev Test 2 (GET)</Button>
+            </div>
+            <div>
+                {questions.map((q, i) => {
+                    switch(q.type) {
+                        case qType.OPEN_ENDED:
+                            return (
+                                <div className="qq-app mv" key={i}>
+                                    <div>{`Question ${i + 1} - Open Ended`}</div>
+                                    <TextField
+                                        className="qq-app mv"
+                                        value={questions[i].question}
+                                        variant="outlined"
+                                        label="Question"
+                                        size="small"
+                                        onChange={e => updateQuestion(i, { ...q, question: e.target.value })}
+                                    />
+                                    <Button
+                                        className="qq-app m"
+                                        variant="contained"
+                                        color="primary"
+                                        onClick={() => addQuestionAtIndex(i)}
+                                    >+</Button>
+                                    <Button
+                                        className="qq-app m"
+                                        variant="contained"
+                                        color="primary"
+                                        onClick={() => deleteQuestion(i)}
+                                    ><DeleteIcon /></Button>
+                                </div>
+                            );
+                        case qType.NUMERICAL:
+                            // @TODO: Add min/max
```

Comment:
Hasn't min/max already been added?

Reply:
Just caught that too haha. I removed that comment in the next commit.